From 4455a13e8d31416dc44af455710fd7a7ff37ec11 Mon Sep 17 00:00:00 2001
From: Daniel-Constantin Mierla <miconda@gmail.com>
Date: Mon, 28 Apr 2014 13:51:56 +0200
Subject: [PATCH] acc: no request is needed for setting end time on dialog
 termination

- callback function cdr_on_end() doesn't use it at all
- reported by Eduardo Lejarreta, FS#256

(cherry picked from commit 8fcdd23aaa49cb6c1ecb34c4c6b4a88078a860f7)
---
 modules/acc/acc_cdr.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/acc/acc_cdr.c b/modules/acc/acc_cdr.c
index 68792e9..3cab762 100644
--- a/modules/acc/acc_cdr.c
+++ b/modules/acc/acc_cdr.c
@@ -569,7 +569,7 @@ static void cdr_on_end( struct dlg_cell* dialog,
                         int type,
                         struct dlg_cb_params* params)
 {
-    if( !dialog || !params || !params->req)
+    if( !dialog || !params)
     {
         LM_ERR("invalid values\n!");
         return;
-- 
2.0.0.rc0

